Modules/System/Azure Queues/src/AzureStorageQueuesImpl.Codeunit.al

```diff
@@ -0,0 +1,293 @@
+        if CheckIfQueueExists(StorageAccountName, Queue) then begin
+            HttpContent.WriteFrom('<QueueMessage><MessageText>' + MessageBody + '</MessageText></QueueMessage>');
+            HttpClient.Post(GetAzureQueueMessagesEndpoint(StorageAccountName, Queue), HttpContent, HttpResponse);
+            exit(HttpResponse.IsSuccessStatusCode);
+        end;
+    end;
+
+    procedure UpdateMessageToQueue(StorageAccountName: Text; Queue: Text; MessageId: Text; PopReceipt: Text; NewMessageBody: Text): Boolean
+    var
+        HttpClient: HttpClient;
+        HttpHeaders: HttpHeaders;
+        HttpContent: HttpContent;
+        HttpRequest: HttpRequestMessage;
+        HttpResponse: HttpResponseMessage;
+        ResponseText: Text;
+    begin
+        if CheckIfQueueExists(StorageAccountName, Queue) then begin
+            HttpContent.WriteFrom('<QueueMessage><MessageText>' + NewMessageBody + '</MessageText></QueueMessage>');
+            HttpClient.Put(GetAzureQueueUpdateMessageEndpoint(StorageAccountName, Queue, MessageId, PopReceipt), HttpContent, HttpResponse);
+            exit(HttpResponse.IsSuccessStatusCode);
+        end;
+    end;
+
+    procedure GetNextMessageFromQueue(StorageAccountName: Text; Queue: Text[20]): Text
+    var
+        HttpClient: HttpClient;
+        HttpHeaders: HttpHeaders;
+        HttpContent: HttpContent;
+        HttpRequest: HttpRequestMessage;
+        HttpResponse: HttpResponseMessage;
+        ResponseText: Text;
+    begin
+        if CheckIfQueueExists(StorageAccountName, Queue) then begin
+            HttpClient.Get(GetAzureQueueMessagesEndpoint(StorageAccountName, Queue), HttpResponse);
+            HttpResponse.Content().ReadAs(ResponseText);
+
+            if HttpResponse.IsSuccessStatusCode then begin
+                Exit(ResponseText);
+            end else
+                Error(ResponseText);
+        end;
+    end;
+
+    procedure PeekNextMessageFromQueue(StorageAccountName: Text; Queue: Text[20]): Text
+    var
+        HttpClient: HttpClient;
+        HttpHeaders: HttpHeaders;
+        HttpContent: HttpContent;
+        HttpRequest: HttpRequestMessage;
+        HttpResponse: HttpResponseMessage;
+        ResponseText: Text;
+    begin
+        if CheckIfQueueExists(StorageAccountName, Queue) then begin
+            HttpClient.Get(GetAzureQueuePeekMessagesEndpoint(StorageAccountName, Queue), HttpResponse);
+            HttpResponse.Content().ReadAs(ResponseText);
+            if HttpResponse.IsSuccessStatusCode then begin
+                Exit(ResponseText);
+            end else
+                Error(ResponseText);
+        end;
+    end;
+
+    procedure GetMessageIdFromResponseBody(var ResponseBody: Text): Text
+    var
+        MessageDocument: XmlDocument;
+        MessageId: XmlNode;
+    begin
+        if XmlDocument.ReadFrom(ResponseBody, MessageDocument) then begin
+            if MessageDocument.SelectSingleNode('QueueMessagesList/QueueMessage/MessageId', MessageId) then begin
```

Comment:
Since this is not a compound statement, an if-then would suffice. This occurs in several procedures in this codeunit.